mcp_zammad/client.py

```
@@ -401,3 +403,633 @@ def get_article_attachments(self, _ticket_id: int, article_id: int) -> list[dict
+                }
+            ],
+        }
+        response = self.api.session.post(self._kb_url(kb_id, "answers"), json=payload)
+        return self._kb_raise_or_return(response)
+
+    def _fill_ids_from_answer(
+        self,
+        answer: dict[str, Any],
+        category_id: int | None,
+        translation_id: int | None,
+        updating_text: bool,
```

Ruff (0.15.4)

[error] Boolean-typed positional argument in function definition

(FBT001)